src/openforms/submissions/admin.py

```diff
@@ -125,7 +122,6 @@ class SubmissionLogInline(GenericTabularInline):
     def get_queryset(self, request):
         qs = super().get_queryset(request)
         qs = qs.exclude_tag(TimelineLogTags.AVG)
-        qs = qs.filter(~Q(template="logging/events/submission_logic_evaluated.txt"))
```

Comment:
we need to keep this line to ignore those log rules in the inline logs view.

I think it'd be good to create a separate issue to add a `prune_logs` management command/celery task (preferably in django-timeline-logger library itself) so we can remove logs. It's a DB table that grows over time (append only now) so we should give administrators the controls to delete logs older than X number of days or logs related to a submission that has since been deleted. So, eventually, over time, all these log records will be deleted and then we'll be able to remove these templates/log records too for full removal.

Reply:
Issue created: https://github.com/maykinmedia/django-timeline-logger/issues/26